x-pack/solutions/security/packages/kbn-scout-security/src/playwright/fixtures/worker/roles_descriptors/index.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { readRolesDescriptorsFromResource, SERVERLESS_ROLES_ROOT_PATH } from '@kbn/es';
+import { ElasticsearchRoleDescriptor, playwrightTest as base } from '@kbn/scout';
+
+export interface RoleDescriptorsFixture {
+  serverless: Map<string, ElasticsearchRoleDescriptor>;
+}
+
+/**
+ * Fixture that provides the role descriptors for the serverless cluster.
+ * Using worker scope to avoid reading the file for each test.
+ */
+export const roleDesciptorsFixture = base.extend<{}, { roleDescriptors: RoleDescriptorsFixture }>({
```

Comment:
might not be the best name, open to renaming :)

Reply:
Nothing comes to my mind right now, I believe is not that bad tbh.